AspNetCore3ChildlifetimeScope/SampleControllerApplicationB.cs

```diff
@@ -0,0 +1,35 @@
+using AspNetCore3ChildlifetimeScope.Dependencies;
+using Autofac;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Hosting;
+
+namespace AspNetCore3ChildlifetimeScope
+{
+    [ApiController]
+    [Route("ApplicationB")]
```

Comment:
Simplify the naming of the controller classes and such so more standard attribute routing can be used. `[Route("api/[controller]")]` or `[Route("[controller]")]` sorts of things. Also, generally controllers end with `Controller` like `ApplicationAController` even if the `[ApiController]` attribute allows you to do without the convention. Keeping folks in familiar territory helps remove questions.